### PR TITLE
Nataliia chobu

### PR DESCRIPTION
**Pull request by nataliiachobu**
Hey guys! Another test for git practice!//learning how to make approvals to merge new changes. ;))

### CODE REVIEW

**Review by mariiasuzdal — APPROVED** [review]
Great job!

**Comment by Nonos1500** [comment]
Looks good, but can you make your methods void, so it would be shorter in main method LOL, just kidding. Everything is good here

**Review by Nonos1500 — APPROVED** [review]

File: src/nataliia_chobu/Divide.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+public class Divide {
+    public static void main(String[] args) {
+        System.out.println(divideWithoutDivisionOperator(12,3));
+    }
+
+    public static int divideWithoutDivisionOperator(int a, int b) {
+
+        int result = 0;
+
```
Comment (AndrewSloth) [inline src/nataliia_chobu/Divide.java:11]: I believe this method should be able work with all numbers. Try dividing -12 / 3 result will be 0 it is incorrect.

**Review by AndrewSloth — CHANGES_REQUESTED** [review]
I believe "divideWithoutDivisionOperator" method should be able work with all numbers. Try dividing -12 / 3 result will be 0 it is incorrect.

**Review by AndrewSloth — APPROVED** [review]

**Pull request merged**